src/components/query-form/query-form.js

```diff
@@ -198,7 +200,7 @@ const QueryForm = props => {
         {successMessage}
 
         <Dropdown className='download-dropdown' disabled={isQueryLoading}>
-          <Dropdown.Toggle title='Download Returned Results' style={{ background: '#00aeef', color: '#f7f7f7', border: 'none', padding: '1rem' }} />
+          <Dropdown.Toggle title='Download Returned Results' style={{ background: downloadButtonColor, color: '#f7f7f7', border: 'none', padding: '1rem' }} />
```

Comment:
This one was annoying. There's a bug in this dropdown library, can only style it like this.